docs/local_setup_bare_metal.md

```diff
@@ -264,9 +264,9 @@ $ ./bootstrap.sh
 
 Please note that sometimes it is needed to fix some "side" errors like
 rebuilding `libyajl` by hands (it is located in `tmp-bootstrap/libyajl`
-subdirectory.
+subdirectory).
 
-Last check if binary has been produced can be made:
+Finally, check if binary has been produced can be made:
```

Comment:
sorry, I meant that the "can be made" is redundant, right?

Reply:
it is, indeed

Reply:
ok second attempt :)